products.py

```diff
@@ -2,30 +2,28 @@
 
 
 class Product:
+    expiration = "expiry date not set"
```

Comment:
class variable name should be uppercased and more meaningfull like `EXPIRY_DATE_NOT_SET` or `EXPIRY_DATE_DEFAULT` or `NOT_SET` (if we wanna treat this variable as more generic and use it for other attributes in the future); consider shortening the value